### ISSUE TITLE
Camera and Webapp: Seperation of Concerns

### ISSUE THREAD

**Issue by kevdouglass** [ChicoState/open-source-security-camera#53]
OsCam #39 

Camera and Webapp: Seperation of Concerns
Currently our Camera lives in our Django web app. We want the user to be able to view the feed of the camera on the web app but the camera functionality need not live there. 

The TASK: 
(1) The task will be to get the camera to work (entirely seperate from our webapp) and when it registers movement & saves the Video file it will have access to our current SQlLite3 database that's shared with Django.
(2) Python Script that creates connection to our SqlLite3 database.
 - the Pi/Camera should be able to save (POST) data to the database upon occurance of Event
 - should be able to access data regarding the user and their contact info (email) so we can send email (see #2 and #45 )

**Comment by melippmann** [comment]
I think this issue should be broken into multiple issues:
- [ ] separation of concerns: web app video stream to monitor live feed and main motion detection program which handles most everything else.
- [ ]  motion detection program should access the database to get settings that are updated from the 'web-app'
- [ ]  main program should access the database to post records of new events (this is being handled by "send_email.py")
- [ ] script that creates connection to database 
- [ ] access database to send emails

**Comment by kevdouglass** [comment]
@melippmann quick question for you and @ConnorGanaway .. Whoever sees this first I guess, 

First, does script that @ConnorGanaway created provide any of the 2 left open checkboxes from above? if so, does it create the entire database or ?? what types of interactions does it have with the database? is it the same database as the SqlLite3 db that our Django models utilize? 

I have some potential fixes for drop-box storage so would be nice to know what I have to work with && I'm also trying to close some of these issues I have open still from last sprint. Thanks

**Comment by melippmann** [comment]
I have made progress on the second check-box 

- [ ] motion detection program should access the database to get settings that are updated from the 'web-app'

meaning technically, if we are just trying to meet semantics, it is done, the main motion detection app is pulling from the shared data-base, but I have not finished hooking all of it up yet.

**Comment by melippmann** [comment]
I don't think we are going to need a script that creates a data-base connection, we are just doing it in the programs where needed.

**Comment by melippmann** [comment]
@kevdouglass I guess, yes, these tasks are both technically done.
